cli/src/commands/extensions/permissions.rs

```diff
@@ -242,7 +246,7 @@ impl From<&Permissions> for PermissionsOptions {
 
         let allow_env = value.env.get().cloned();
         let allow_net = value.net.get().cloned();
-        let allow_run = value.run.get().cloned();
+        let allow_run = value.unsandboxed_run.get().cloned();
```

Comment:
Does this replace `run` with `unsandboxed_run` because those `run` permissions are actually managed by Deno?

Reply:
It just renames the `run` permissions used by deno in our manifest to `unsandboxed_run` to make it a little more clear these do not use any sandboxing.